src/github-app-setup.js

```diff
@@ -51,6 +51,12 @@ async function obtainAppToken (id, privateKeyInput) {
   }
 }
 
+const configureAppGit = (token) => {
+  const command = `git config --global url."https://x-access-token:${token}@github.com/".insteadOf "https://github.com/"`
+  info(`App > ${command}`)
```

Comment:
Is this going to reveal the access token on console output?

Reply:
Github will mask it out should it be logged thanks to `setSecret` - https://github.com/daspn/private-actions-checkout/blob/master/src/github-app-setup.js#L45

Reply:
Awesome! Thanks for clarifying!